openslides_votecollector/views.py

```diff
@@ -187,6 +187,17 @@ def no_error_context(self):
         """
         return {}
 
+    @staticmethod
```

Comment:
It could also be a normal instance method. We only use it in a view instance context, don't we. See https://www.webucator.com/blog/2016/05/when-to-use-static-methods-in-python-never/.